pkg/virt-handler/options.go

```diff
@@ -15,13 +15,15 @@ func virtualMachineOptions(
 	capabilities *api.Capabilities,
 	disksInfo map[string]*containerdisk.DiskInfo,
 	expandDisksEnabled bool,
+	memoryOverCommitmentEnabled bool,
```

Comment:
nit:
for the record, I think we need to refactor this so it would generate the right options without providing all of these parameters. For example, it could check by itself which feature gates are active. That would save duplication and make things much more elegant. In any case, it's outside of this PR's scope.

Reply:
Totally agree 🙂